pkg/tlsutil/tlsutil.go

```diff
@@ -127,3 +128,21 @@ func NewSignedCertificate(cfg CertConfig, key *rsa.PrivateKey, caCert *x509.Cert
 	}
 	return x509.ParseCertificate(certDERBytes)
 }
+
+// NewRandomToken generates an n-byte random hexadecimal token. The output will have length n*2.
+func NewRandomToken(n int) ([]byte, error) {
```

Comment:
nit: there's actually a defined format for another authenticator. We might just want to copy that for now: https://kubernetes.io/docs/admin/bootstrap-tokens/#token-format